README.md

```diff
@@ -29,9 +29,13 @@ curl -L https://raw.githubusercontent.com/egold/better-xcode-ibeam-cursor/master
 
 ###### For Xcode 7.3 and later:
 
-1. Clone this repository
+1. Clone this repository (the lines below assume you've cloned to your home directory)
 2. Create a backup of `/Applications/Xcode.app/Contents/SharedFrameworks/DVTKit.framework/Resources/Assets.car`
-3. Copy (`sudo cp`) the Assets.car file to `/Applications/Xcode.app/Contents/SharedFrameworks/DVTKit.framework/Resources/Assets.car`
+3. Patch the Assets.car file with the appropriate patch:
+```
+cd /Applications/Xcode.app/Contents/SharedFrameworks/DVTKit.framework/Resources/Assets.car
```

Comment:
Did you mean `cd /Applications/Xcode.app/Contents/SharedFrameworks/DVTKit.framework/Resources` here? Thanks for the patch, btw!